app/lib/html_css_to_image.rb

```diff
@@ -14,13 +14,15 @@ def self.url(html:, css: nil, google_fonts: nil)
 
   def self.fetch_url(html:, css: nil, google_fonts: nil)
     cache_key = "htmlcssimage/#{html}/#{css}/#{google_fonts}"
+    # We could remove this after some days :)
     cached_url = Rails.cache.read(cache_key)
 
     return cached_url if cached_url.present?
 
     image_url = url(html: html, css: css, google_fonts: google_fonts)
 
     Rails.cache.write(cache_key, image_url) unless image_url == FALLBACK_IMAGE
+    RedisRailsCache.write(cache_key, image_url) unless image_url == FALLBACK_IMAGE
```

Comment:
Because I want this to bake for at least a few days(likely a week) and don't want these keys to expire too quickly let's add an `expires_in: 2.weeks` to this line. That is probably overkill but I want to make sure we have enough time to build up the Redis cache keys. After a week I will peek into Redis and see how many we have and then try to figure out if we can handle flipping over completely to it.